src/ensembl/src/shared/components/layout/StandardAppLayout.test.tsx

```diff
@@ -65,54 +66,57 @@ const minimalProps = {
 };
 
 describe('StandardAppLayout', () => {
-  afterEach(() => {
+  beforeEach(() => {
```

Comment:
Think most of our test cases use `afterEach` instead of `beforeEach`. Shouldn't we stick with just one of them?

Reply:
Yeah. The reason I am switching to beforeEach is because I think react-testing-library might do its cleanups a bit too late to be caught in afterEach. I remember one of the tests failing in a genome browser component when I was doing the cleanup in afterEach rather than in the beforeEach.

Reply:
That's interesting. So we'll gradually need to change the test cases to use `beforeEach` then 🤔